code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -496,6 +511,21 @@
 			return 0
 	return 1
 
+/mob/living/simple_animal/proc/attack_threshold_check(damage, damagetype)

```

Comment:
might want to link me to it,m mine looks the same as the live version there..unless its a PR then i need to look harder..i also just compeletly fail at finding things at times.